client/my-sites/backup/backup-storage-space/style.scss

```
@@ -0,0 +1,30 @@
+.backup-storage-space {
+	margin-top: 1rem;
+	padding: 16px 32px;
+	background: #fff;
+	box-shadow: 0 0 0 1px #dcdcde;
+}
+
+.backup-storage-space__progress-bar-container {
+	display: grid;
+	grid-template-columns: max-content auto max-content;
+	align-items: center;
+
+	@include breakpoint-deprecated( '<480px' ) {
+		grid-template-columns: none;
+		grid-template-rows: max-content auto max-content;
+	}
+}
+
+.backup-storage-space__progress-bar {
+	padding: 0 24px;
+
+	@include breakpoint-deprecated( '<480px' ) {
+		padding: 0;
+	}
+}
+
+.backup-storage-space__divider {
+	margin: 16px -32px;
+	border-bottom: 1px solid var( --studio-gray-10 );
```

Leaving a comment here to mark one more spot we could potentially replace with a theme-derived color:

```suggestion
	border-bottom: 1px solid var( --color-neutral-10 );
```